### PR TITLE
Vending machines go brrrr - Tealove and others request

### PR DESCRIPTION
**Pull request by JamieD1**
Vending machines go brrrr - Tealove and others request

In short they space every room on the station, they deal too much damage, just not a fun event,

Also low pop its just like cancer.

#### Changelog

:cl:    
rscdel: Disabled Vending machine sentience event
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
you're given plenty forewarning when theyre going to become sentient, right?

all it takes is one dude with a screwdriver and crowbar...

**Comment by JamieD1** [comment]
> you're given plenty forewarning when theyre going to become sentient, right?
> 
> all it takes is one dude with a screwdriver and crowbar...

every machine?.. You nuts?

**Comment by ynot01** [comment]
> > you're given plenty forewarning when theyre going to become sentient, right?
> > all it takes is one dude with a screwdriver and crowbar...
> 
> every machine?.. You nuts?

no, actually, if players want to keep their department they should pick up 2 simple tools and recon the vendors

this contradicts my earlier statement but it holds true that one guy with a screwdriver and crowbar can cover over half the stations vendors just by going to eva, center hallway, sec hallway, engie hallway, and arrivals

if they ignore it like many do then they pay for it

i always find myself laughing when i prepare my department and everyone else is screaming wHy VEndor blOw uP???? HElp

**Comment by jcatjmeow** [comment]
Jamie you know in the template where it says "intent of your pull request"

You're supposed to explain the intent of your pull request there

**Comment by JamieD1** [comment]
> > > you're given plenty forewarning when theyre going to become sentient, right?
> > > all it takes is one dude with a screwdriver and crowbar...
> > 
> > 
> > every machine?.. You nuts?
> 
> no, actually, if players want to keep their department they should pick up 2 simple tools and recon the vendors
> 
> this contradicts my earlier statement but it holds true that one guy with a screwdriver and crowbar can cover over half the stations vendors just by going to eva, center hallway, sec hallway, engie hallway, and arrivals
> 
> if they ignore it like many do then they pay for it
> 
> i always find myself laughing when i prepare my department and everyone else is screaming wHy VEndor blOw uP???? HElp

But they hurt

**Comment by JamieD1** [comment]
> Jamie you know in the template where it says "intent of your pull request"
> 
> You're supposed to explain the intent of your pull request there

Better?

**Comment by Jarod1200** [comment]
They're supposed to hurt and it is really easy to prevent them from being a problem if you screwdriver and crowbar them. If you want to make them hurt a lot less, make their explosion not cause a hull breach

**Comment by Hopekz** [comment]
Stop trying to turn our codebase into hugbox

**Comment by JamieD1** [comment]
> Stop trying to turn our codebase into hugbox

Difference between hugbox and stupidbox.

**Comment by Marmio64** [comment]
rather have this event have less chance to occur, this event is RAMPANT brand intelligence, decrease its chance and increase brand intelligence, which only makes this happen to one vendor i believe.

And yeah, its not that hard to fix the vendors and it helps spice the round up, this ALREADY only happens like once every day.

File: code/modules/events/brand_intelligence.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/brand_intelligence
 	name = "Brand Intelligence"
 	typepath = /datum/round_event/brand_intelligence
-	weight = 5
+	weight = 0
```
Comment (Hopekz) [inline code/modules/events/brand_intelligence.dm]:
```suggestion
	weight = 0 // I hate fun
```

**Comment by overlord12345** [comment]
Fun fact if you deconstruct the original vendor the event is stopped, its legit that easy. The fact someone thinks its too destructive is fucking mental

**Pull request closed without merging**